devops/README.md

```diff
@@ -0,0 +1,34 @@
+# Deployment
+
+## Images building + Jenkins
+
+The deployment of images and updating of the services and applications is performed by a Jenkins
+workflow found [here](https://jenkins01.tacc.utexas.edu/view/Hazmapper+Geoapi/.)
+
+The images used in deployment are built automatically for the master branch using TravisCI and 
+pushed to Docker Hub (see https://hub.docker.com/r/taccaci/hazmapper).
+
+## Access and Troubleshooting
+The configurations of each of the following are in their associated directories. 
+
+### VM for nginx:  
+
+    hazmapper.tacc.utexas.edu 
+    
+    see hazmapper repo [ngnix.conf] (https://github.com/TACC-Cloud/hazmapper/blob/master/nginx.conf)
+
+### VMs for backend services 
```

Comment:
Do we want to include the `experimental.geo-services.tacc.utexas.edu` host in this list?

Reply:
I'm suggesting in https://github.com/TACC-Cloud/geoapi/pull/183#discussion_r1516419656 to not list the actual vms. I'm thinking if we just list the general types (geoapi-services, geoapi-workers...) and then a link to the confluence.